src/Autocomplete.js

```diff
@@ -63,29 +62,32 @@ class Autocomplete extends Component {
       m,
       l,
       offset,
+      minLength,
       ...props
     } = this.props;
 
     const _id = 'autocomplete-input';
-    const _autocomplete = 'autocomplete';
     const _ul = 'autocomplete-content dropdown-content';
+    const sizes = { s, m, l };
+    let classes = {};
+    constants.SIZES.forEach(size => {
+      classes[size + sizes[size]] = sizes[size];
+    });
 
     return (
       <Row>
-        <Col s={s} m={m} l={l} offset={offset} className={cx('input-field', className)} {...props}>
+        <Col offset={offset} className={cx(cx('input-field', className), classes)} {...props}>
           {icon && this.renderIcon(icon, iconClassName)}
-          <input type="text" id={_id} className={_autocomplete} onChange={this._onChange}/>
+          <input type="text" id={_id} className='autocomplete' onChange={this._onChange} />
           <label htmlFor={_id}>{title}</label>
           <ul className={_ul}>{this.renderDropdown(data)}</ul>
```

Comment:
Sorry to be a pain, but I really don't think we need a variable for `_ul`, also could we move the entire UL inside {renderDropdown} ?